src/pages/ProjectView.js

```diff
@@ -0,0 +1,65 @@
+import { useCallback, useEffect } from "react"
+import { useSelector, useDispatch } from "react-redux"
+import { useNavigate, useParams } from "react-router-dom"
+import { getProject, reset } from "../features/project/projectSlice"
+import DefaultContainer from "../layout/DefaultContainer"
+import ContentContainer from "../layout/ContentContainer"
+import { Title } from "../elements/Title"
+import { Button } from "../elements/Button"
+import { REACT_APP_BE_HOST } from "../config/index"
+
+function Project() {
+  const navigate = useNavigate()
+  const dispatch = useDispatch()
+  const { id: currentProjectId } = useParams()
+
+  useEffect(() => {
+    dispatch(getProject(currentProjectId))
+    return () => {
+      dispatch(reset())
+    }
+  }, [navigate, dispatch, currentProjectId])
+  const { project } = useSelector((state) => state.project)
```

Comment:
Let's move it up to other inline hooks which returns value. It's good practice to group hooks which returns value, hooks which doesn't return anything ( like useEffect ), handlers